migrating_from_ocp_3_to_4/premigration-checklists.adoc

```diff
@@ -0,0 +1,90 @@
+
+toc::[]
+
+Before you migrate your application workloads with the {mtc-full} ({mtc-short}), review the following checklists.
+
+[id="source-cluster-checklist_{context}"]
+== Source cluster checklist
+
+* [ ] The cluster meets the link:https://docs.openshift.com/container-platform/3.11/install/prerequisites.html#hardware[minimum hardware requirements].
+* [ ] The {product-title} version is 3.7, 3.9, 3.10, 3.11.
+* [ ] The {mtc-short} version is the same on all clusters.
+* [ ] All nodes have an active {product-title} subscription. This will avoid issues if you require customer support.
+* [ ] All the link:https://docs.openshift.com/container-platform/3.11/day_two_guide/run_once_tasks.html#day-two-guide-default-storage-class[run-once tasks] have been performed.
+* [ ] All the link:https://docs.openshift.com/container-platform/3.11/day_two_guide/environment_health_checks.html[environment health checks] have been performed.
+* [ ] You have checked for persistent volumes (PVs) with abnormal configurations  stuck in a *Terminating* state by running the following command:
++
+[source,terminal]
+----
+$ oc get pv
+----
+
+* [ ] You have checked for pods whose status is other than *Running* or *Completed* by running the following command:
++
+[source,terminal]
+----
+$ oc get pods --all-namespaces | egrep -v 'Running | Completed'
+----
+
+* [ ] You have checked for pods with a high restart count by running the following command:
++
+[source,terminal]
+----
+$ oc get pods --all-namespaces --field-selector=status.phase=Running \
+  -o json | jq '.items[]|select(any( .status.containerStatuses[]; \
+  .restartCount > 3))|.metadata.name'
+----
++
+Even if the pods are in a *Running* state, a high restart count might indicate underlying problems.
+
+* [ ] The internal container image registry uses a link:https://docs.openshift.com/container-platform/3.11/scaling_performance/optimizing_storage.html#registry[supported storage type].
+* [ ] You can read and write images to the registry.
+* [ ] The link:https://access.redhat.com/articles/3093761[etcd cluster] is healthy.
+* [ ] The link:https://docs.openshift.com/container-platform/3.11/install_config/master_node_configuration.html#master-node-configuration-node-qps-burst[average API server response time] on the source cluster is less than 50 ms.
+* [ ] The cluster certificates are link:https://docs.openshift.com/container-platform/3.11/install_config/redeploying_certificates.html#install-config-cert-expiry[valid] for the duration of the migration process.
+* [ ] You have checked for pending certificate requests by running the following command:
++
+[source,terminal]
+----
+$ oc get csr -A | grep pending -i
+----
+
+* [ ] The link:https://docs.openshift.com/container-platform/3.11/install_config/configuring_authentication.html#overview[identity provider] is working.
+
+[id="target-cluster-checklist_{context}"]
+== Target cluster checklist
+
+* [ ] The {mtc-short} version is the same on all clusters.
```

Comment:
Similar to the above comment, should this say that the version is the same as the source cluster?